NEWS.md

```diff
@@ -38,6 +38,22 @@
 
 ## Features and fixes
 
+* `_lgl()`, `_int()`, `_int()`, and `_dbl()` now use the same (strict) coercion
+  methods as vctrs (#904). This means that:
+  
+    * `map_chr(TRUE, identity)`, `map_chr(0L, identity)`, and 
+      `map_chr(1L, identity)` are deprecated because we now believe that 
+      converting a logical/integer/double to a character vector should require 
+      an explicit coercion.
+      
+    * `map_dbl(1.5, identity)` now fails because we believe that silently 
+      truncating doubles to integers is dangerous. But note that 
+      `map_dbl(1, identity)` still works since no numeric precision is lost.
```

Comment:
Isn't this `map_int()` in both examples here?

Reply:
Ooops yes.